### ISSUE TITLE
Interactivty not working with MVT source

### ISSUE THREAD

**Issue by alrocar** [CartoDB/carto-vl#1263]
We have a viewer for the Tilesets server where the Interactivity is working with v0.8.0 but it does not work for v1.1.1

Here's a minimal example:

```
<!DOCTYPE html>
<html>
<head>
    <meta charset='utf-8'/>
    <title>CARTO VL – Vector tiles – Polygons | CARTO</title>
    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no'/>
    <script src="https://libs.cartocdn.com/carto-vl/v1.1.1/carto-vl.min.js"></script>
    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.js"></script>
    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.css" rel="stylesheet" />
    <style>
        body {
            margin: 0;
            padding: 0;
        }
        #map {
            position: absolute;
            top: 0;
            left: 0;
            width: 100%;
            height: 100%;
        }
    </style>
</head>
<body>
<div id='map'></div>
<script>
    let config = {
        map: {
            container: 'map',
            style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
            center: [0, 0],
            zoom: 0,
            maxzoom: 14,
        },
        controls: {

        }
    };

    const map = new mapboxgl.Map(config.map);

    const properties = {
        id: {
            type: 'category'
        }
    };

    const source = new carto.source.MVT('http://do-tilesets-1-st.azew.cartodb.net:8889/api/v1/data-observatory/census/{z}/{x}/{y}.mvt?api_key=API_KEY&measurement=demographics.mrli', {
        idProperty: 'id',
        'properties': properties
    });
    const viz = new carto.Viz(`
        color: #000
        strokeColor: #000A
        strokeWidth: 1
    `);
    const layer = new carto.Layer('test', source, viz);
    const interactivity = new carto.Interactivity(layer);
    interactivity.on('featureClick', featureEvent => {
        if (featureEvent.features.length) {
            const feature = featureEvent.features[0];
            const vars = feature.variables;
            console.log(`
                ID: ${feature.id}
            `);
        }
    });

    layer.addTo(map, 'watername_ocean');
</script>

</body>
</html>
```

The MVT source comes from a staging server (so you need VPN connection).

Also ask me for the `API_KEY` needed in order this example to work.

I have a similar example using one of my CARTO maps vector tiles URL (instead of the tilesets server) and the interactivity is working... so I'm not sure what is happening.

cc. @VictorVelarde for triaging

**Comment by Algunenano** [comment]
`featurePieces` reaches [getCompoundFeature](https://github.com/CartoDB/carto-vl/blob/2e48a09aafce90d46c682391ac083a9fa6f35883/src/interactivity/commonFeature.js#L21) as `[]`, which makes `exemplar` === undefined.

I don't know if the error in not considering that possibility or reaching that function with an empty array, though.

**Comment by VictorVelarde** [comment]
I have just checked it and the fix was already merged to master some time ago, here: https://github.com/CartoDB/carto-vl/pull/1240, so it will be included in the next release (probably a 1.1.x patch soon).

I wanted to be sure that this was exactly the same issue as that one, as we've had some MVT source issues recently.

**Comment by alrocar** [comment]
It works in master!

Thanks @Algunenano and @VictorVelarde for the quick responses 🙇
